antarest/study/storage/variantstudy/model/command/remove_renewables_cluster.py

```diff
@@ -71,6 +66,10 @@ def _apply_config(
             )
         self._remove_renewables_cluster(study_data)
 
+        for binding in study_data.bindings:
+            if f"{self.area_id}.{self.cluster_id}" in binding.clusters:
+                study_data.bindings.remove(binding)
+
```

Comment:
add this in _remove_renewables_cluster method.
Also effectively remove the binding constraint data in the _apply method

Reply:
you may want to mutualize the code found in remove_cluster which remove a binding constraint given some cluster id (and maybe the one found in remove_area for area ids)